addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -1900,6 +1900,10 @@ export function moveNodes(
 // Prepare / Save / Restore state utilities
 //------------------------------------------------------------------------------
 
+const prepareUpdateLockedEditables = new Set();
+const lockEditablePrepareUpdate = editable => editable && prepareUpdateLockedEditables.add(editable);
+const unlockEditablePrepareUpdate = editable => editable && prepareUpdateLockedEditables.delete(editable);
+const isEditablePrepareUpdateLocked = editable => editable && prepareUpdateLockedEditables.has(editable);
```

Comment:
Why ? Those are litteraly just one self-explanatory statement protected by one simple check each, and they are even used only once. Is this some kind of test to see if I'd spot them ?

Reply:
I think I had tried something less self explanatory first and then kept the general form. I don't dislike this because of readability, I don't need to think about what these do. But it's true it's overkill :-P